Makefile

```
@@ -0,0 +1,23 @@
+build:
+	mvn package -DskipTests=true
+
+clean:
+	mvn clean
+
+test:
+	mvn test
+
+server: build
+	java -jar target/*.jar --server --ebnf ./ysql_grammar.ebnf --debug
```

`*` seems like bad practice: it could pick up multiple files

---

For now, there is only one jar file and it has the version number in its name. so some form of matching is needed.

---

could it be like `rrdiagram-*.jar`?

---

not sure why it was marked resolved

```suggestion
	java -jar target/rrdiagram-*.jar --server --ebnf ./ysql_grammar.ebnf --debug
```